.github/workflows/ci2.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       DOCKER_BASE: ${{ matrix.base }}
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
a major version change like this can often mean an API change but there doesn't appear to be one here.

When changing from v2 to v3 the default depth changed from everything to 1 which caused a few scripts to fail in interesting ways.

Reply:
FWIW, some GHA were already on v4. So hopefully nothing too breaking from v3 to v4